test/specs/point.spec.js

```diff
@@ -1,92 +1,55 @@
 describe('Point annotation', function() {
   describe('auto', jasmine.fixtures('point'));
 
-  const eventIn = function(xScale, yScale, element) {
-    const options = element.options;
-    const adjust = options.borderWidth / 2 - 1;
-    return {x: element.x, y: element.y - element.height / 2 - adjust};
-  };
-  const eventOut = function(xScale, yScale, element) {
-    const options = element.options;
-    const adjust = options.borderWidth / 2 + 1;
-    return {x: element.x, y: element.y - element.height / 2 - adjust};
-  };
-
-  window.testEvents({
-    type: 'point',
-    id: 'test',
-    xValue: 5,
-    yValue: 5,
-    radius: 30,
-    borderWidth: 12
-  }, eventIn, eventOut);
-
-  describe('events, removing borderWidth by callback, ', function() {
-    const chartConfig = {
-      type: 'scatter',
-      options: {
-        animation: false,
-        scales: {
-          x: {
-            display: false,
-            min: 0,
-            max: 10
-          },
-          y: {
-            display: false,
-            min: 0,
-            max: 10
-          }
-        },
-        plugins: {
-          legend: false,
-          annotation: {
-            annotations: {
-              point: {
-                type: 'point',
-                xValue: 5,
-                yValue: 5,
-                radius: 20,
-                borderWidth: 10
-              }
-            }
-          }
-        }
-      },
+  describe('inRange', function() {
+    const annotation = {
+      type: 'point',
+      xValue: 5,
+      yValue: 5,
+      radius: 30
     };
 
-    const pointOpts = chartConfig.options.plugins.annotation.annotations.point;
-
-    it('should detect click event', function(done) {
-      const clickSpy = jasmine.createSpy('click');
+    const chart = window.scatter10x10({test: annotation});
+    const element = window.getAnnotationElements(chart)[0];
 
-      pointOpts.click = function(ctx) {
-        if (ctx.element.options.borderWidth) {
-          delete ctx.element.options.borderWidth;
-          ctx.chart.draw();
-        } else {
-          ctx.element.options.click = clickSpy;
+    it('should return true inside element', function() {
```

Comment:
This could be improved a little, testing 8 points around the point center, so an rectangle would not pass. Same for the "outside" test.

Reply:
It's not clear to me what "so an rectangle would not pass." mean. Here we are testing a point (whatever point style will be applied, the rule of inRange is always related to the circle).

Reply:
![image](https://user-images.githubusercontent.com/27971921/150413412-e46fbe15-fe86-413c-876b-46636b2dd494.png)

A (bad) picture is better than 1000 words, they say. :)
Testing the points marked with blue arrows does not detect if the inRange returned true in those gray areas.

Reply:
done

Reply:
I use the same angles as you did for the ellipses. ;)